.env.dusk.ci

```diff
@@ -1,52 +0,0 @@
-APP_ENV=ci
```

Comment:
@DFurnes To make sure that things in `.env.dusk.ci` and `.env.dusk.local` got ignored in github, I tried `git rm -rf --cached .` and then `git add .` It then just deleted those files all together here - is this ok?

Reply:
We probably don't want to delete the files altogether since they set other useful test environment settings (like using the file cache driver), but we can provide specific overrides by exporting environment variables to the shell (from protected environment variable in the [Wercker config](https://app.wercker.com/dosomething/rogue/environment)).

Reply:
Although that still leaves us with trouble on local dev, doesn't it. 😖 Hmm, maybe we do gitignore these files but then use a `.env.dusk` template to create them?

Reply:
would this just to be creating `.env.dusk.ci.example` and .env.dusk.local.example` files?

Reply:
Yeah, that's what I was thinking. Still a little awkward, but better than nothing!